test.py

```diff
@@ -1,13 +1,84 @@
-from sympy import sympify, symbols
 
-# Get user input for a mathematical expression
-user_input = input("Enter a mathematical expression: ")
+import numpy as np
+import sympy as sp
+import math as m
+
+t, y = sp.symbols("t y")
+ode = t + y -1
+
+def EulersMethod(t_n,h,y0,ode):
+    for i in t_n:
+        yn = y0 + h * ode.subs({t:i,y:y0})
+        y0 = yn
+    return yn
```

Comment:
Please move this function to its own file(e.g. euler.py). test.py is for unit tests only